File: MODULE 3/3A_JAVASCRIPTS/JAVASCRIPTS CODING CHALLENGE/Task9.js
// JAVASCRIPT CODING CHALLENGE

// 9. Return the number of vowels in a string.
const letters = ("Learning Javascript to upskill in Automation Testing")

function vowelsNum(string) {
    let Vowels = "aAeEiIoOuU";
    let vowelsCount = 0;
    
    for (let num = 0; num < string.length; num++) { 
        if (Vowels.includes(string[num])){
            vowelsCount++;
        }
    }
    return vowelsCount;    
}
console.log(vowelsNum(letters))

File: MODULE 4C/PYTHON_CODING_CHALLENGE/Task9.py
"""
PYTHON CODING CHALLENGE
9. Return the number of vowels in a string.
"""
def count_vowels(s):
    vowels = "aeiouAEIOU"
    count = 0
    for char in s:
        if char in vowels:
            count += 1
    print("The number of vowels in the string is:", count)
    return count

